L1Trigger/DTTrigger/src/DTTrigTest.cc

```diff
@@ -194,11 +194,9 @@ void DTTrigTest::beginJob() {
 }
 
 void DTTrigTest::beginRun(const edm::Run& iRun, const edm::EventSetup& iEventSetup) {
-  if (!my_trig) {
```

Comment:
Looks to me like they meant to do
```C++
if(my_trig) {
```

Reply:
`my_trig` is defined as `new` in the constructor, and only deleted in the destructor: I tend to agree with @iarspider that this check is probably useless
@battibass ?

Reply:
If the existence of the `my_trig` is guaranteed as an invariant of the class, then I agreee there is no need for the check. 

Looking at the code, it seems to me like `my_trig` should just have been a 
```C++
DTTrig my_trig;
```
rather than 
```C++
DTTrig* my_trig;
```
would would have enforced the guarantee and made memory management easier.